### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by dlwub**
In this activity:
- A route has been created in the route file
- Corresponding actions of the route have been added to the controllers
- View files have been created for the corresponding actions

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @dlwub , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

- [ ] Please update the different headers (`Setup, Install, Usage`) under the `Getting Started`  section of your README file. This will guide recruiters to be able to set up your project and test it.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -81,17 +81,16 @@ In order to run this project you need:
...
+  cd Rails-blog
 
 ### Install
 
 Install this project with:
 
 Example commands:
-cd blog-app
+cd Rails-blog
   gem install
```
Comment (Gambit142) [inline README.md:96]: - [ ] The link to the LICENSE in the README file of this project directs people to the template article for creating a valid LICENSE. I would suggest that you read the article and create a valid LICENSE for this project. The instructions for creating the LICENSE can be found under the _**How to apply this license**_ heading in the template. 👍 😎 

Screenshot 2023-04-03 182023

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @dlwub  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ Used the correct Git Flow 👍 
✔️ It has corresponding view files for controllers 
✔️ Routes are restful 

## Required Changes ♻️
- [ ]  Check the comments below 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @dlwub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**